lardataobj/RecoBase/TrackingDicts/classes_def.xml

```diff
@@ -43,6 +43,39 @@
    <version ClassVersion="10" checksum="3286658271"/>
   </class>
 
+    <!-- schema evolution rules -->
+      <!-- version 11 -->
+        <!-- * positions -->
+  <ioread 
+    version="[-10]" 
+    sourceClass="recob::Trajectory"
+    source="std::vector<ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<double>, ROOT::Math::GlobalCoordinateSystemTag>> fPositions;"
+    targetClass="recob::Trajectory"
+    target="fPositions"
+    include="Math/GenVector/CoordinateSystemTags.h;Math/GenVector/Cartesian3D.h;Math/GenVector/PositionVector3D.h">
+    <![CDATA[
+      fPositions.clear();
```

Comment:
Calling clear is not necessary, both here and below.

Reply:
Probably. I did not check the context this snipped is inserted too: I can imagine `fPositions` is a local variable just default-constructed, but I went for the safest approach nevertheless.
After all, I am effectively setting a private data member from outside the class, so I don't want to assume standard rules apply.

Reply:
After talking with Philippe, it turns out that the calling clear is the safest thing to do.  So I withdraw my comments regarding it.